src/test/java/reposense/parser/CsvParserTest.java

```diff
@@ -247,4 +249,23 @@ public void repoConfig_defaultBranch_success() throws ParseException, IOExceptio
         Assert.assertEquals(expectedConfig.getBranch(), actualConfigs.get(0).getBranch());
         Assert.assertEquals(expectedConfig.getBranch(), authorConfigs.get(0).getBranch());
     }
+
+    @Test
+    public void repoConfig_overrideKeyword_success() throws ParseException, IOException, HelpScreenException {
```

Comment:
Do add on to the other `repoConfig` test, to check that the 'override' value for the elements are `false`